components/MashCards/ContactCard.tsx

```diff
@@ -0,0 +1,60 @@
+import s from './MashCard.module.scss';
+
+interface Props {
+  clientName: string;
+  timeLeft: string;
+  dateTime: string;
+  referrer: string;
+  requestedSupport: string;
+}
+const ContactCard = ({
+  clientName,
+  timeLeft,
+  dateTime,
+  referrer,
+  requestedSupport,
+}: Props): React.ReactElement | null => {
```

Comment:
Why are we saying this can return null? To me it seems that it would always return a React.ReactElement.

Same applies to our other card components.

Reply:
Not actually sure tbh - it might have happened if I accidentally hit tab?

What does ' | null ' mean?

Reply:
| represents or. so : React.ReactElement | null means we are stating function returns a react component or null.

However, looking at the code I think it only ever returns a react component so we can remove the | null.